bitwarden_license/src/CommCore/Services/ProviderService.cs

```diff
@@ -120,8 +120,7 @@ public async Task UpdateAsync(Provider provider, bool updateBilling = false)
             await _providerRepository.ReplaceAsync(provider);
         }
 
-        public async Task<List<ProviderUser>> InviteUserAsync(Guid providerId, Guid invitingUserId,
```

Comment:
I believe the reason `invitingUserId` was required was to do the permission check on the service end. I know the API endpoint has this validation, but adding it to the service method helps ensure integrity further on if we decide to reuse this method.

Reply:
Would it not make more sense to just add invitingUserId back as an optional parameter later if we decide to use it in the way you are describing?

Reply:
My concern with making that optional is we WANT it to be used. Admin project is a rare exception. I'd rather make the code more self-documenting by making a lame bypass parameter than requiring reading the code to understand why `invitingUserId` is optional.

If you feel differently, that's OK. please just include xml comments with an appropriate description of the parameter and when it's OK to leave default and to always set if you can.

Reply:
I'm more trying to say that right now this parameter is unreferenced. If we decide to do permissions validation in the service at some point wouldn't we solve how to pass data around then? 

I used an optional parameter as a solution example just because that is what we do right now for organization invites.

Reply:
Oh I think we should validate in the service in this PR too. Sorry, I probably should have explicitly said that.

I didn't realize org invites had optional inviting user Ids. I guess that's the way we're going right now. I'd still like xml comment so intellisense can warn you of the proper use